backend/core/test/listings/listings.e2e-spec.ts

```diff
@@ -70,6 +70,11 @@ describe("Listings", () => {
     expect(res.body.items.map((listing) => listing.id).length).toBeGreaterThan(0)
   })
 
+  it("should return all external + internal listings", async () => {
```

Comment:
A few more things here

* The title of this unit test does not describe what this test does, please update

* Note that tests are generally supposed to test smaller units of code that can be logically isolated (https://smartbear.com/learn/automated-testing/what-is-unit-testing/#:~:text=A%20unit%20test%20is%20a,of%20the%20definition%20is%20important.).

* You should test the functionality that you added. Suggestions for tests: 
  * `it("should return the same response as /listings",...`
  * `it("should pass its own ListingQueryParams through to list()",...`

As this feature gets fleshed out these tests will continue to be useful.

Note that some of these tests may make more sense in `listings.service.spec.ts`, not sure what the delineation is.